docs/release-notes/2020.04-release-notes.md

```diff
@@ -0,0 +1,201 @@
+gniBS5aSc2pGdl3UNqSx+CcyS+/Jrn4LDnPHj3mx7WGlJLc/Lw0uYZxSZM3FAj7/
+zrsLgnP4/xdcCMTcbVbC6L7VBSIE+hE6ppdrKwoLp7C2cyA14DnHM+heqx8Dqtf6
+acIc8txyUB7zShGC5Fhp37YnhrHGnUrDxxYi99OfNChXAbwtR5z8Leicijplc/NF
+oibGSLLC8NvFkzuWIIuqZK8l2/UZmER7Nt7hhHszilsnAJenwvYCIVEqJsrdsTUf
+iiYm5AifwhU018Ia5As9kvBqymMy2bYSbZZVic+WXb6StWHugjKSFJmCo2h66QsI
+giHNJEvOxybANlSkxI6t38EipGvvW1dlROBuL/RZOQvSJL7SRlclOeQF4ZtPBMFy
+yI5p3DBZbIHChdaH5OE0U7zk+qt1hvO+HSh8Yc1OVBd6pziAay5L6S8jsBKDPjJr
+7OTUuHJL00teBe3SORS8BACkg6+/6Yk9a3lSiQARAQABiQIlBBgBCgAPBQJZB65r
+AhsMBQkFo5qAAAoJEDoCGvrb4xiHUqcP/iOUf55a2rU5N/tlZsn2fd+O6U2V4Iym
+YYysJ5OZJj0baDmL+SSmNarem+ogRcpBmKX3LFNgGnEoRVaeILQApri5JelCSEhC
+Ex5Xwk1WOqXTRWV9flbVU/rTSiCNDUySaaJDCVOQ07TEQ/bDShkR0X+MyCzklgbZ
+rZnj+spaPeDnpIkfwfG2aKiJ0hq9KATBwdREcP9f/R7LaAdmcbr89Ya4iPEmaDYO
+gh/t83zH45XKiWxXwb9rn9U2szWLOfx8rMTTRhYWBPB+K7RX8mGoiphqtMOJYhNA
+N7AVm+9LRVxRcnmWyf1cQKF8FG1j9FBlaeNUAFefpvvcBffTFvPnpqOaSZm7nu6l
+NwWT2N0IHdr9daET0ewvL2poMkE9P3abPmEjQYFMobg3ZiPs7e5TeplvVFtC2y89
+47Gow54KIVS2cbeSVCPlCj/B8GmEjkAsGmjDtGTk45LiJptv6h5lH95YGUafI97U
+xLSEJJ+mab1iSRqDCn8lVlfjvAp6Hv/DW9sr6lutQuzZjxhFi5rsly1C40rkOni/
+pByJDW47+VERp2wxvJdVVUZEge53IHTgrruHs/tWUsJNEswkHQ7OsY5u7bEcPUlW
+Stua8WJo5U4cN0abgrRAXnXXBWi2bNbYe8xnH+OsOghb1gRg3N99G/TthoziPQ1I
+cuYxZQalebJS
+=fC+/
+-----END PGP PUBLIC KEY BLOCK-----
+```
+
+How to verify:
+
+* Download `<ISO>.sig` file from mirror
+* Run: `gpg2 --verify <ISO>.sig <ISO>`. You should see the following output:
+
+```
+    gpg: Signature made November  5, 2019 at 05:40:09 PM UTC
+    gpg:                using RSA key F5E984E5ADD29578D45CC1FE3A021AFADBE31887
+    gpg: Good signature from "OpenIndiana Release Engineering" [ultimate]
+```
+
+  Even if the output includes:
+
+```
+    gpg: WARNING: This key is not certified with a trusted signature!
+    gpg: There is no indication that the signature belongs to the owner.
+```
+
+  The signature is still correct.
+
+## General system changes
+
+All OpenIndiana system software was rewritten in Python 3.
+Installation images now don't deliver Python 2.7 interpreters or libraries, however some software still requires Python 2.7.
+We've moved to GCC 7 as the default compiler.
+
+## Desktop software and libraries
+
+* LibreOffice 6.4.3 was added. There are some issues with GTK 3 theme, so "gen" VCL plugin is used by default. You can experiment with GTK3 plugin, setting SAL_USE_VCLPLUGIN environment variable to gtk3 explicitly.
+* VirtualBox was updated to 6.1.6. There are some issues with direct and VNC consoles with non-EFI guests.
+* VLC was updated to 3.0.10
+* ntfsprogs were updated to 2017.3.23AR.5
+* 32-bit versions of basix xorg tools were dropped
+* hplip was updated to 3.19.12
+* rhythmbox was updated to 3.4.4
+* Gstreamer was updated to 1.16.2
+* Due to serious performance issues with totem>3.0.1, we can't update totem to the latest version. Switching to own totem fork allowed to import upstream patches and remove dependency on Python 2.7.
+* UPower was updated to 0.99.11
```

Comment:
I would state it: "We created our own totem fork in order to  import upstream patches and remove the dependency on Python 2.7."

Reply:
When you omitted "our" from my text you should have added "an" in front of "own". (We created an own totem fork) or (We created a totem fork)